docs/common/install-istioctl-tool.md

```diff
@@ -8,7 +8,9 @@ operators to debug and diagnose Istio service mesh deployments.
 
 Use an `istioctl` version that is the same version as the Istio control plane 
 for the Service Mesh deployment. See [Istio Releases](https://github.com/istio/istio/releases) for a list of valid 
-releases, including Beta releases.
+releases, including Beta releases. 
+
+These instructions assume that the Sail Operator has installed `Istio` in the `istio-system` namespace.
```

Comment:
Sorry if this is overly pedantic but `Istio` is a cluster-wide resource that is not namespace scoped. When I see `Istio` with backticks I assume you are talking about the `Istio` resource and if it's Istio without backticks I assume you're talking about Istio the project.

Maybe say something like the "Sail Operator has installed an `Istio` in your cluster? I don't think the namespace matters in this context. In fact, you can remove the namespace from step 2. below e.g.
```sh
$ oc get istio
```

Reply:
Good catch - thank! I had forgotten that `Istio` is a cluster wide resource, and the `-n istio-system` below implied to me that the resource must exist in the "istio-system" namespace, which is not guaranteed if it were namespace scoped. Based on what you said, I think it's probably best to just remove the `-n istio-system` part of the command below and I'll delete this addition. WDYT?

Reply:
@longmuir yep that sounds good! Sorry I missed your question until just now.